activestate.yaml

```diff
@@ -46,14 +46,6 @@ constants:
     if: ne .OS.Name "Windows"
     value: .sh
 scripts:
-  - name: install-deps
-    language: bash
-    if: ne .Shell "cmd"
-    value: |
-      if ! type "gozip" &> /dev/null; then
```

Comment:
This was originally needed for the offline-installer, but that was moved to its own repo, so this is no longer needed.